scripts/requirements-fixed.txt

```diff
@@ -1,8 +1,8 @@
 #
-# This file is autogenerated by pip-compile with Python 3.10
+# This file is autogenerated by pip-compile with Python 3.8
```

Comment:
This file was added in previous commit, using pip-compile with Python 3.10.

Now, in this commit, in same pr, we are suddenly running pip-compile with Python 3.8.

Why has the Python version used for creating `requirement-fixed.txt` suddenly changed ?

And could this change impact the outcome of pip-compile ?

I would like only a single commit in this PR to have the requirements-fixed.txt to be updated, cause I believe that some of the changes to requirements-fixed.txt between commit  https://github.com/nrfconnect/sdk-nrf/pull/12193/commits/7851b12da6f944c7fbeb9d749f66f788f25fc2b5 and https://github.com/nrfconnect/sdk-nrf/pull/12193/commits/0176f8f4850d169ea72017d23702480c6845179d

Some of the reason the second commit suddenly has changes may be because of:
- zbor is no longer having a fixed revision
- Some pip-packages may have been updated remotely, so that a new version is now fetched, compared to when PR was initially opened.
-  Switch from python 3.10 to python 3.8

Reply:
python set to 3.8
zbor vesrion is not fixed
commits related with python packages squashed to one commit
rebased on top of main branch
we will work with autocommit future on separate PR